modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/commands/sql/SqlReplCommand.java

```diff
@@ -5,43 +5,53 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
-import java.util.concurrent.Callable;
+import org.apache.ignite.cli.call.configuration.ReplCallInput;
+import org.apache.ignite.cli.commands.decorators.TableDecorator;
+import org.apache.ignite.cli.core.call.DefaultCallExecutionPipeline;
+import org.apache.ignite.cli.core.repl.executor.SqlReplCommandExecutor;
 import org.apache.ignite.cli.sql.SqlManager;
-import org.apache.ignite.cli.sql.table.Table;
 import picocli.CommandLine.Command;
+import picocli.CommandLine.Model.CommandSpec;
 import picocli.CommandLine.Option;
+import picocli.CommandLine.Spec;
 
 /**
  * Command for sql execution.
  */
 @Command(name = "sql")
-public class SqlReplCommand implements Callable<Table<String>> {
+public class SqlReplCommand implements Runnable {
```

Comment:
Should it be called SqlReplCommand? I think 'repl' here is out of date.

Reply:
Yes, and similarly SqlReplCommandExecutor should be called SqlQueryCall. I have this in local changes and will probably include with the completer.